packages/worker/src/entities/transactionReceipt.entity.ts

```diff
@@ -34,8 +34,8 @@ export class TransactionReceipt extends CountableEntity {
   @Column({ type: "int" })
   public readonly type: number;
 
-  @Column({ type: "bytea", transformer: hexTransformer })
-  public readonly root: string;
+  @Column({ type: "bytea", transformer: hexTransformer, nullable: true })
+  public readonly root?: string;
```

Comment:
Maybe makes sense leaving a comment that this field is only relevant on L1 pre byzantium fork ([EIP658](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-658.md)), but was present in ZKsync API before, so it cannot be just dropped.

Reply:
good idea, done.